Cargo.toml

```diff
@@ -3,6 +3,10 @@ name = "portablegabi-pallet"
 version = "0.3.0"
 authors = ["KILT <info@kilt.io>"]
 edition = "2018"
+repository = "https://github.com/KILTprotocol/portablegabi-pallet"
+homepage = "https://github.com/KILTprotocol/portablegabi"
```

Comment:
I would maybe link to our portablegabi tutorial? But this is also fine.

Reply:
Agree, will change.